File: Partie-Backend/src/main/java/com/basketapp/basket_app/models/Document.java
package com.basketapp.basket_app.models;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Lob;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Document {
	
	 @Id
	 @GeneratedValue (strategy = GenerationType.IDENTITY)
	 private  Long idDocument;
	 private String titre;
	 
	 @Lob
     byte[] contenu;  
}

File: Partie-Mobile/lib/models/Document.dart
class Document {
  int? idDocument;
  String? titre;
  String? contenu;

  Document({this.idDocument, this.titre, this.contenu});

  Document.fromJson(Map<String, dynamic> json) {
    idDocument = json['idDocument'];
    titre = json['titre'];
    contenu = json['contenu'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['idDocument'] = this.idDocument;
    data['titre'] = this.titre;
    data['contenu'] = this.contenu;
    return data;
  }
}